### ISSUE TITLE
Auto-complete forms causing freeze in microsoft edge chromium

### ISSUE THREAD

**Issue by rmcpantoja** [nvaccess/nvda#12911]
### Steps to reproduce:

1. open microsoft edge (chromium version).
2. open a page in which we have suggestions for autofilling in forms or edit fields.
3. Press a letter not before activating NVDA's focus mode in the edit box.
4. Finally is where the error occurs that freezes the browser and NVDA and unfreezes for approximately one minute.

### Actual behavior:

Due to the above, both the browser and NVDA freeze when writing in certain edit boxes that have autocomplete suggestions or any suggestions.

### Expected behavior:

As in chrome, I would hope that there would be no freezing issues when focusing and trying to type in a text box. This in chrome does not happen, since you can write anyway.

#### NVDA installed/portable/running from source:

Installable version

#### NVDA version:
2021.2
#### Windows version:
Windows 10 21H2 (x64) build 19044.1200
#### Name and version of other software in use when reproducing the issue:
Microsoft Edge - version 94.0.992.38
#### Other information about your system:

* ram 4 gb
* four CPU cores
* HP
* Windows 10 64 bit.
#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
This has been going on for a year, after Microsoft Edge Chromium was released to the whole world. Since version 2020.1 is where I went through this problem and it was the exact same behavior.
#### If NVDA add-ons are disabled, is your problem still occurring?

Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes, I feel and think that it is an internal problem.

#### conclusion

In summary, microsoft edge presents crashes in some edit boxes with suggestions and what it does exactly is freeze the browser for approximately one minute with the current versions of Microsedge, Windows and NVDA, even without the plugins or addons.

**Comment by gigicop** [comment]
I have exactly the same issue with my HP laptop with 16Gb of ram and the same version of Microsoft Edge. NVDA 2021.2 with Espeak and other synthesizers freezes for about a minute in almost every login form after typing one or two letters. By disabling the autofill option it works fine. On my Lenovo desktop it only freezes for a couple of seconds.
          Thanks.

**Comment by brunoprietog** [comment]
This also happens to me sometimes when copying URLs to the clipboard from the address bar.

**Comment by rmcpantoja** [comment]
> This also happens to me sometimes when copying URLs to the clipboard from the address bar.

Yes, it happens to me too but it is very little. As I mentioned in the first post, it happens more often in the fields. Hope the problem is fixed, because currently edge is the browser that I use very frequently.

**Comment by rogergoll** [comment]
version  104.0.1293.1 l,  Linux Mint 20.3 Cinnamon, GeForce GTX 1050 nvidia , 23 gb ram .  I have the same problem with form fields or autocomplete and the problem continues even when I disable the graphics acceleration. To continue using the browser it is necessary to disable any profile data or payment profile autocomplete etc.

**Comment by cary-rowen** [comment]
I can reproduce this too, is there anything new with NV Access on this?

**Comment by seanbudd** [comment]
I cannot reproduce this problem with NVDA 2022.2beta2 and:
* the latest Edge stable `Version 103.0.1264.44 (Official build) (64-bit)`.
* the latest Edge canary `Version 105.0.1309.0 (Official build) canary (64-bit)`

After filling out my personal information fully in edge settings `edge://settings/personalinfo`, I tested this with a form that auto-filled all personal information details.

@cary-rowen / @rogergoll  - can you confirm:
* the version of Edge you are using
* the version of NVDA you are using
* and also provide an example website with a form which causes the freeze/crash

Additionally, Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

Kind Regards,
NV Access Software Developers

**Comment by seanbudd** [comment]
Is anyone able to reproduce this and give the requested information in the previous comment?

cc @bhavyashah @cary-rowen

**Comment by rafaeldomi** [comment]
I'm having this issue too. Edge: Version 104.0.1293.5
Linux 5.11.11-051111-generic #202103310025 SMP Thu Apr 1 01:54:13 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
Ubuntu 20.04
Site: https://www.confluent.io/online-talk/best-practices-for-apache-kafka-in-production-confluent-online-talk-series/

It happens in version 105, I have downgraded to 104, but didnt solve. Before i upgraded to 105 version i didnt experience any problems.

**Comment by seanbudd** [comment]
@rafaeldomi - can you please explain the Linux Version information? NVDA is software for Windows.

**Comment by seanbudd** [comment]
@rafaeldomi - The registration form on that page does not freeze for me using NVDA 2022.2rc1 and Edge 103.0.1264.49

As requested in the previous comment, to address this we also need:
- the version of NVDA you are using
- a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

**Comment by seanbudd** [comment]
Without further information, closing as stale / can't reproduce.

If the required information is provided, please open a new issue or comment the information and this issue will be reopened.
